OnlTools/Jevp/StJevpBuilders/fttBuilder.h

```diff
@@ -2,23 +2,208 @@
 #include <stdlib.h>
 
 #include "JevpBuilder.h"
-class daqReader;
+#include "DAQ_READER/daqReader.h"
```

Comment:
This. You don't need to include the whole shebang if the forward declaration is enough.